src/epjson/epJSONTranslator.cpp

```diff
@@ -179,12 +195,13 @@ std::string fixupEnumerationValue(const Json::Value& schema, const std::string&
   }
 
   if (fieldType == openstudio::IddFieldType::RealType) {
-    const auto& objectProperties = getSchemaObjectProperties(schema, type_description);
-    const auto& field = safeLookupValue(objectProperties, fieldName, "anyOf");
-    if (field.isArray()) {
+
+    const auto& fieldProperty = getSchemaObjectFieldProperty(schema, type_description, group_name, field_name, "anyOf");
```

Comment:
This will correctly get the `anyOf` of regular fields and extensible fields alike (when group_name is not empty)